pkg/cmd/miscellaneous.go

```diff
@@ -158,34 +154,6 @@ func getLoggingCredentials() (username, password string) {
 	return username, password
 }
 
-// getSeedNamespaceNameForShoot returns namespace name
-func getSeedNamespaceNameForShoot(shootName string) (namespaceSeed string) {
```

Comment:
I am add one `lookupTargetInfo` method today, It will always full set up information in memory for query, it didn't care about target.Stack()[1].Kind either project or seed. besides, it maybe more flexible instead of using target.Stack()[x].Name

just one concern from line +386 to +402 it may takes a time to query, without this part it will more light to lookup 
The advantage for this part +386 to +402 is full set up information for lookup query.  I will continue check other part tomorrow. welcome any feedback.